error.go

```diff
@@ -14,6 +14,7 @@ const (
 	ErrorTypeInvalidRequest ErrorType = "invalid_request_error"
 	ErrorTypePermission     ErrorType = "more_permissions_required"
 	ErrorTypeRateLimit      ErrorType = "rate_limit_error"
+	ErrorTypeIdempotency    ErrorType = "idempotency_error"
```

Comment:
@merrickfox Thanks for the contribution! Very minor, would you mind just moving this up a few lines so that it's sorted alphabetically?

Reply:
Cool, sorted both statements.